docs/devel.md

```diff
@@ -70,6 +70,17 @@ $ podman build -t localhost/coreos-assembler . --from quay.io/coreos-assembler/c
 This is especially useful for changes to dependencies or for final verification
 of COSA code.
 
+## Developing on kola, ore and plume
+
+If you are only doing changes to kola, ore or plume, you can build, test and use them directly with:
+
+```
+$ cd mantle
+$ ./build kola ore plume
+$ ./test
+$ ./bin/kola ...
```

Comment:
This is assuming they have build tools in their host environment (or that they'll layer them). Should the instructions instead say to do this from inside a `cosa shell`? (And from there it's not a far leap to the "hack inside a container" workflow. :) )

Reply:
Sure, you can do that from a toolbox too. I'll add a note.